pages/admin/reports.tsx

```diff
@@ -4,13 +4,113 @@ import { Text, GridItem } from '@chakra-ui/react'; // Chakra UI
 import Layout from '@components/admin/Layout'; // Layout component
 import { Role } from '@lib/types'; // Role enum
 import { authorize } from '@tools/authorization'; // Page authorization
+import { Box, Flex, Input, Button, Spinner } from '@chakra-ui/react'; // Chakra UI
+import { DownloadIcon, SearchIcon } from '@chakra-ui/icons'; // Chakra UI icons
+import { useState } from 'react'; // React
 
 // Internal home page
 export default function Reports() {
+  // const { dateRange, addDayToDateRange, dateRangeString } = useDateRangePicker();
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
```

Comment:
```suggestion
  const [startDate, setStartDate] = useState(undefined);
  const [endDate, setEndDate] = useState(undefined);
```

Reply:
I think I need to keep these as quotations to work, or else an error comes up where string cannot be assigned to parameter type of undefined. We also do this in the AdminModal, EditUserInformationModal, and AssignNumberModal. Let me know what you think.